src/vOptGeneric.jl

```diff
@@ -51,26 +51,32 @@ function vModel(; args...)
     return m
 end
 
-function vSolve(m::JuMP.Model ; relax=false, method=nothing, step = 1., round_results = false, verbose = true, args...)
+function vSolve(m::JuMP.Model ; relax=false, method=nothing, verbose = true, kwargs...)
 
     vd = getvOptData(m)
 
     if relax != false
         @warn "linear relaxation not yet implemented"
     end
-    
-    if method == :epsilon
-        solve_eps(m, step, round_results, verbose ; relaxation=relax, args...)
-    elseif method == :dicho || method == :dichotomy
-        solve_dicho(m, round_results, verbose ; relaxation=relax, args...)
-    elseif method == :Chalmet || method == :chalmet
-        solve_Chalmet(m, step, verbose ; relaxation=relax, args...)
-    elseif method == :lex || method == :lexico
-        solve_lexico(m, verbose ; relaxation=relax, args...)
+
+    methods = Dict{Symbol, Function}([
+        :epsilon => solve_eps,
+        :dicho => solve_dicho,
+        :dichotomy => solve_dicho,
+        :chalmet => solve_Chalmet,
+        :Chalmet => solve_Chalmet,
+        :lexico => solve_lexico,
+        :lex => solve_lexico,
+    ])
+
+    if method in keys(methods)
+        solve_function = methods[method]
+        solve_function(m; verbose=verbose, relaxation=relax, kwargs...)

```

Comment:
`step` and `round_result` are passed through `kwargs`, so dispatch is not necessary anymore. This permits to reduce the "cost" of adding new methods by just extending the `Dict`.